ipam/api.go

```diff
@@ -115,19 +105,17 @@ func (h *Handler) initMux() {
 	h.HandleFunc(capabilitiesPath, func(w http.ResponseWriter, r *http.Request, p map[string]string) error {
 		res, err := h.ipam.GetCapabilities(r.Context())
 		if err != nil {
-			sdk.EncodeResponse(w, NewErrorResponse(err.Error()), true)
 			return err
 		}
-		sdk.EncodeResponse(w, res, false)
```

Comment:
Now that errors are encoded in a middleware, `sdk.EncodeResponse` is only called in case of success. Hence the third argument is no longer needed.